multipath-formula/multipath/defaults.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
in the file /etc/multipath.conf there is defaults part that define general behavior of the service:
 we are using for example this:
 defaults {
  path_grouping_policy multibus
  path_checker tur
  polling_interval 15
  no_path_retry queue
  checker_timeout 60
}
but of course there is a lot of parameters for tuning see `man multipath.conf` 
would be good place to have it in this yaml file of some other place so it can be easily change once we need it

Reply:
Hi,

`defaults.yaml` just contains default values for the formula to use when rendering the configuration, they can all be overwritten and expanded using the `multipath` pillar.
Do you think it makes sense to add the configuration options you listed here, to the formula defaults?
Keep in mind that out of the box our multipath-tools package does not ship with any `multipath.conf` file.
The one `device` block for NetApp I added to the formula defaults because I figured it is the same on all our deployments and rarely, if ever, changes.

Reply:
As discussed today, I amended the formula defaults.